aries-site/src/pages/templates/global-banner-notifications.mdx

```diff
@@ -0,0 +1,161 @@
+import { Anchor } from 'grommet';
+import {  BestPracticeGroup, Example } from '../../layouts';
+import {
+  BannerContentLayoutExample,
+  BannerNotificationCritical,
+  BannerNotificationCriticalClose,
+  BannerNotificationDoExample,
+  BannerNotificationDontExample,
+  BannerNotificationInfo,
+  BannerNotificationWarning,
+  BannerNotificationWarningClose,
+  BannerNotificationDiagram,
+} from '../../examples';
+
+
+<Example
+  showResponsiveControls={['fullScreen']}
+  // height, width, and screenContainer.scale value chosen to best visually 
+  // represent the example
+  height="650px"
+  width="100%"
+  caption="An example of a dashboard with a Global Banner warning the user that their subscription is expiring. The user can either dismiss the notification or click on the anchor to renew their subscription."
+  screenContainer={{ scale: 0.57 }}
+>
+  <BannerContentLayoutExample />
+</Example>
+
+
+## Guidance
+Global Banners are high-attention notifications used primarily for status updates on background processes and upcoming system changes and releases.
+
+### Usage
+- Global Banners are used to showcase Warning and Critical statuses or informational alerts. Depending on the severity of the information, they may remain persistent across multiple sessions or be dismissible by the user.
+- Since Global Banners take over the top of an interface and can be intrusive, they should be reserved only for important system generated updates.
+- Also when applicable, Global Banners should always have a relevant anchor(s) to direct users to further actions or to view more about the issue in full detail.
+
+### Do Not Use Global Banners For
+- Completion of tasks, such as long running or batch actions.
+- Success or failure confirmations.
+- Low-attention and low-severity information.
+  - For low-severity notifications that do not require user action or user attention such as confirmation receipts, instead use [Toast Notifications](templates/toast-notifications).
```

Comment:
```suggestion

For low-severity notifications that do not require user action or user attention such as confirmation receipts, instead use [Toast Notifications](templates/toast-notifications).
```

Reply:
@taysea good suggestion. Also, did you want me to commit these suggestions or leave that to you and Brittany?

Reply:
If you're good with the copy suggestion, go for it! @KennyAtHPE